app/helpers/api/v1/books_helper.rb

```diff
@@ -0,0 +1,6 @@
+module Api
```

Comment:
You can delete this file

Reply:
Done

Reply:
👀

Reply:
@aleochoam I deleted all helpers files, those was generated with the controller.